font.go

```diff
@@ -1,89 +1,100 @@
 package figure
 
 import (
-  "bufio"
-  "strings"
-  "io"
+	"bufio"
+	"bytes"
+	"io"
+	"path"
+	"strings"
 )
 
 const defaultFont = "standard"
 
 type font struct {
-  name string
-  height int
-  baseline int
-  hardblank byte
-  reverse bool
-  letters [][]string
+	name      string
+	height    int
+	baseline  int
+	hardblank byte
+	reverse   bool
+	letters   [][]string
 }
 
 func newFont(name string) (font font) {
-  font.setName(name)
-  file := getFile(font.name)
-  defer file.Close()
-  scanner := bufio.NewScanner(file)
-  font.setAttributes(scanner)
-  font.setLetters(scanner)
-  return font
+	font.setName(name)
+	fontBytes, err := Asset(path.Join("fonts", name+".flf"))
```

Comment:
This line needs to be `fontBytes, err := Asset(path.Join("fonts", font.name+".flf"))` in order for the default font to work.  That is, using `name` instead of `font.name` undoes the work of the line above.